### PR TITLE
added scatter data

### PR DESCRIPTION
**Pull request by priyanka-cshl**

### CODE REVIEW

**Comment by priyanka-cshl** [comment]
please merge - best scatter plot ever

File: somePlots.m [hunk 1]
```diff
@@ -10,10 +10,11 @@
 handles.Axes.Line = axes('Position', [.15 .5 .3 .2]);
 title('Line plot')
 
-
 %% A scatter plot
 handles.Axes.Scatter = axes('Position', [2*.15+.3 .5 .3 .2]);
 title('Scatter ploot')
+scatter(rand(10,1),rand(10,1))
```
Comment (tsgouvea) [inline somePlots.m]: Why 10? 15 would be so much better.
What do you guys think?

Comment (priyanka-cshl) [reply somePlots.m]: true...

Comment (iuliuv) [reply somePlots.m]: 13?

**Pull request merged**